sdk/tests/features/feature_extractor.py

```diff
@@ -0,0 +1,52 @@
+import importlib
+import inspect
+import pkgutil
+
+from features.get_project_root import get_project_root
+
+
+def validate_feature_list(feature_list):
+    for feature in feature_list:
+        if feature == "":
+            raise ValueError("lusid_feature error: Some decorated methods have no value passed. "
+                             "Please make sure each lusid_feature decorator has a value code passed.")
+
+    unique_features = set(feature_list)
+    if len(unique_features) != len(feature_list):
+        sorted_features = sorted(feature_list)
+        for i, code in enumerate(sorted_features):
```

Comment:
if the above check passes do you need to do this?

Reply:
This is optional, but a nice to have because not only it tells you that a feature code is a duplicate, but also it tells you exactly which code number is a duplicate, making it much easier to find

Reply:
I wonder if it might be a nice feature to actually allow duplicates, there could be more than one example which satisfies given feature

Reply:
If there were two different implementations of the same feature, we would need to find a way how to display that in the feature matrix. I'm thinking there still would have to be some extra parameter to differentiate the two? Could be simpler to use sub-codes for different feature implementations eg. F1-1, F1-2, and reflect that in the yaml file. That way we also wouldn't have to re-architecture the whole reporter

Reply:
but of course let's go with what will work best in the long term